Controller/AudioPlayer.cs

```diff
@@ -76,9 +76,13 @@ private static void ClearSongQueue()
         }
 
         public static void PlayPlaylist(Playlist playlist, int startIndex = 0)

```

Comment:
Is het nie handiger om deze functie weg te halen en de andere aanpassen naar dat je dus een playlist meegeeft

Reply:
Wat bedoel je?
Deze functie krijgt al een playlist. Alleen vanuit de mock db, kun je geen context gebruiken.
Dus heb ik 't aangepast, zodat je ook alleen nummers mee kan geven.

Daarom dus, twee functies. En de een roept de ander aan.

Reply:
In PlayPlaylist_SongQueue_ContainsSongs() word alleen nog maar die andere functie gebruikt (Playlistsongs meegeven), dus beter toch om er een functie van te maken waarbij een Playlist meegeeft

Reply:
Ja dat kan dus niet haha, zo werkt die mock db niet.